pkg/sriov/sriov.go

```diff
@@ -225,6 +225,18 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, net
 			}
 		}
 
+		// reset MTU for VF device until if the MTU was captured in the cache
+		if conf.OrigVfState.MTU != 0 {
+			logging.Debug("Reset VF device MTU",
```

Comment:
Print out the desired MTU value (`conf.OrigVfState.MTU`).

Reply:
done